### ISSUE TITLE
[Feature Request] Improve Mouse's Lines' Selection

### ISSUE THREAD

**Issue by Jack1789** [SubtitleEdit/subtitleedit#3743]
Hi, I'm actually wondering why it isn't possible to simply select multiple subtitle's lines while pressing and holding the mouse's right button? I mean, currently only one can be selected at a time with it  and, if more need to be selected, one has to obligatorily combine it with the keyboard (shift, ctrl keys) or simply use the keyboard only.

Thanks.

**Comment by OmrSi** [comment]
You can do that with the left button by starting the selection from outside the last column:
![image](https://user-images.githubusercontent.com/20923700/84598529-129deb00-ae74-11ea-94e1-e9c3fb3b3b4b.png)

**Comment by Jack1789** [comment]
Hi, thanks for your reply.

I just noticed there was an extra empty space over there, which allows to do precisely what I have talked about. The thing is, such space is too narrow and it gets kind of annoying having to hit specifically that screen area each time; besides, although the "Text" column can be resized, its width will not be remembered after restarting the program.

So, I bet making the program remember that column's size would fix the issue.
